mmv1/third_party/terraform/services/sql/resource_sql_database_instance_test.go

```diff
@@ -1587,6 +1587,27 @@ func TestAccSQLDatabaseInstance_sqlMysqlDataCacheConfig(t *testing.T) {
 	})
 }
 
+func TestAccSQLDatabaseInstance_sqlPostgresDataCacheConfig(t *testing.T) {
```

Comment:
Let's add one failure test case for enterprise edition.

Reply:
Done